css/style.css

```diff
@@ -1,4 +1,224 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding-top: 24px;
+}
+
+.header__logo-img {
+/* width: 166px; */
+}
+
+.menubar {
+  display: flex;
+}
+
+.menubar__link {
+  text-decoration: none;
+  line-height: 18px;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+}
+
+.menubar__link:hover, .menubar__link:active, .menubar__link:focus {
+  cursor: pointer;
+  color: #E51937;
+  box-shadow: 0 2px 0 #E51937;
+  opacity: .75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+margin-left: 258.14px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-left: 63.14px;
+}
+
+.menubar__link:nth-child(3){
+  margin-left: 24.86px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-left: 44px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-left: 46px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-left: 221.14px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-left: 30px;
+}
+
+.button:nth-child(8) {
+  margin-left: 30px;
+}
+
+.button {
+  display: block;
+  background-color: #ffffff;
+  border: non;
+  width: 24px;
+  height: 15px;
+  position: relative;
+}
+
+.button .button__line, 
+.button .button__line:before,
+.button .button__line:after {
+  position: absolute;
+  display: block;
+  top: 50%; margin-top: -1px;
+  left: 50%; margin-left: -12px;
+  width: inherit;
+  height: 2px;
+  background-color: #E51937;
+}
+
+.button__line:before, .button__line:after {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".button__line:before" to come before selector ".button .button__line:before" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)